### ISSUE TITLE
Add DBKind='U' Database to database list

### ISSUE THREAD

**Issue by h-tsunenaga** [ScriptPup/sqltools-teradata-driver#11]
It seems that DBKind='U' Databases are not listed in database list after connected.
I want this feature to be same as Teradata Studio.

**Comment by ScriptPup** [comment]
Hi there, I'm not familiar with Teradata Studio. 

Currently this is the query that's used to search tables:
```
const searchTables: IBaseQueries["searchTables"] = queryFactory`
SELECT 
  TableName AS "label"
  ,DataBaseName AS "parent"
FROM dbc.tablesV
WHERE 
  TableKind IN ('T','V')
  AND DataBaseName = '${(p) =>
    splitSearch(ContextValue.TABLE, p.search)["database"]}'
;
${(p) => {
  console.log(p);
  return "";
}}
`;
```

Are you requesting that the 'U' be added to this list? 

I'll do some testing to see what that includes by default. If there are a lot of 'U' tables which are non-value add to the majority of users then it may make sense to make this a preference of some kind.

**Comment by ScriptPup** [comment]
Other projects have been of higher priority, especially without response on this one. 

However, I've been looking the remaining feature requests over here and this one seems like it likely isn't possible without some changes to the parent sql-tools project. The ContextValues made available are:
<img width="170" alt="image" src="https://user-images.githubusercontent.com/6163646/233799026-90309c40-ecb5-4894-88fd-803a261d514b.png">

The 'U' table type appears to be specifically for user defined types, which doesn't really fit into this paradigm.
I think in this case the scope of the tool may not match up with this request.

This isn't really intended to be a replacement for administrative type tools like Teradata Studio, Nexus, etc. It's really just intended to be a lightweight and convenient plugin to allow users to run a query real-quick from the same tool they're already using for other work.

I'm going to close this one as not planned for now. That being said, if someone has a good enough argument for this being included and wants to submit a PR, please feel free!

**Comment by ScriptPup** [comment]
It looks like the documentation I found about what 'U' means was misleading... It says U - User-Defined table type. Looks like more appropriate should have been "User object"... I guess that's what I get for trying to research with google instead of just looking at the database.

Ref:
https://support.teradata.com/community?id=community_question&sys_id=e11883271b97fb00682ca8233a4bcb83

I'm going to be implementing a change for #16 which will solve for this as well. We'll have a toggle for "Show user DBs"

**Comment by ScriptPup** [comment]
Implemented. Will be published with 0.4.0 package.

<img width="645" alt="image" src="https://user-images.githubusercontent.com/6163646/233805095-11f9fb84-0d72-4cb4-a34d-143b0b558803.png">


<img width="251" alt="image" src="https://user-images.githubusercontent.com/6163646/233804676-ffab995a-0782-4818-8582-dc5279f31c66.png">
